src/state/actions/regular.ts

```diff
@@ -62,8 +62,11 @@ type RegularActions = {
   'messages/markSeen/end': undefined;
 
   'buddies/completed': Result<typeof buddyApi.fetchBuddies>;
-  'buddies/ban/start': { buddyId: string };
-  'buddies/ban/end': E.Either<string, buddyApi.Buddy>;
+  'buddies/changeStatus/start': {
+    buddyId: string;
+    status: 'Banned' | 'Active';
```

Comment:
banStatus: 'Banned' | 'NotBanned'.

isBanned: true | false?

Reply:
will keep the BanStatuses (Banned | NotBanned) because soon we will have a third option, Archived aswell.

Reply:
![image](https://user-images.githubusercontent.com/34128180/118759592-fe2f5300-b879-11eb-9c27-63550c62e307.png)